app/styles/gavant-ember-validations.scss

```diff
@@ -0,0 +1,20 @@
+.input-validator {
+    &.is-invalid {
+        .custom-select,
+        .form-control  {
+            border-color: #dc3545;
+        }
+        .form-check-label {
+            color: #dc3545;
```

Comment:
we should make this color a variable, say, `$ember-validations-invalid-color`, and put it in a `styles/gavant-ember-validations/_variables.scss` file (like power-select does) and append `!default` to the end of the var declaration, so it can be easily overridden by the app.

Might be worth adding variables for the `min-height` and `padding-top` values below too.